tests/test_command.py

```diff
@@ -0,0 +1,60 @@
+import mock
+
+from django.core.management.base import CommandError
+from django.core.management import call_command
+
+from tests.base import BaseTest
+
+
+def exec_cmd(**kwargs):
+    call_command('hawkrequest', **kwargs)
+
+
+class UnauthorizedResponse:
+    def __init__(self):
+        self.headers = {}
+        self.text = 'Unauthorized'
+
+
+class AuthorizedResponse:
+    def __init__(self):
+        self.headers = {
+            'Server-Authorization': 'xyz',
+            'Content-Type': 'text/plain'
+        }
+        self.text = 'Authorized'
```

Comment:
This looks like it might be easy enough with a [real Response](http://docs.python-requests.org/en/master/api/#requests.Response). If it's not easy then faking it is in fine but this is worth a try:

```python
response = Response()
response.headers['Server-Authorization'] = 'xyz'
response.headers['Content-Type'] = 'text/plain'
response._content = 'Authorized'
```

Reply:
Simpler, more realistic, and works great. Thank you.